WalletWasabi/Hwi/Parsers/HwiParser.cs

```diff
@@ -90,7 +90,7 @@ public static bool TryParseError(JToken token, [NotNullWhen(true)] out HwiExcept
 		return error is not null;
 	}
 
-	public static bool TryParseErrorCode(JToken codeToken, out HwiErrorCode code)
+	public static bool TryParseErrorCode(JToken? codeToken, out HwiErrorCode code)
```

Comment:
There is only a single call of this method. Imho it should look like this and you should make sure that you do not pass `codeToken = null` to this method:

```cs
	public static bool TryParseErrorCode(JToken codeToken, [NotNullWhen(true)] out HwiErrorCode? code)
	{
		code = default;

		try
		{
			var codeInt = codeToken.Value<int>();
			if (Enum.IsDefined(typeof(HwiErrorCode), codeInt))
			{
				code = (HwiErrorCode)codeInt;
				return true;
			}
		}
		catch
		{
			return false;
		}

		return false;
	}
```